src/components/Themes.tsx

```diff
@@ -0,0 +1,75 @@
+import { PaletteMode, ThemeOptions } from "@mui/material"; // TypeScript Types
+// An interface that defines object parameters
+interface ThemeColor {
+    default: string;
+    paper: string;
+    text: string;
+}
+
+//Toggle the color scheme between light and dark mode
+export const baseTheme = (mode: PaletteMode) => {
+    let themeColor: ThemeColor;
+    if (mode === "light") {
+        themeColor = {
+            default: "#F3FFDE",
+            paper: "#F3FFDE",
+            text: "#191919",
+        };
+    } else {
+        // Defind the dark theme color
+        themeColor = {
+            default: "#232726",
+            paper: "#232726",
+            text: "#F3FFDE",
+        };
+    }
+    // allow configuration using `theme`
+    const theme: ThemeOptions = {
+        typography: {
+            fontFamily: ["Nunito Sans", "sans-serif"].join(","),
```

Comment:
This doesn't need to be an array. It can just be written directly as a string. A string will actually be more efficient as the current implementation just creates a string from the array. Since there are only a few values an array really doesn't add any value.